temp/tc_repos/orgs/%25-tCore-test-data-repos.json

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
Should temp files be included?

Reply:
No, will clean this up before PR merge. Also looks like Windows build is now failing to build due to content error. Seeing <br> in file path. Looks like new lines in tn tsv?

Reply:
Could you share more details on this, how is it failing? Don't know I understand what you mean with new lines in tn tsv?